lib/peep.rb

```diff
@@ -0,0 +1,32 @@
+require 'pg'
+
+class Peep
+  def self.create(content:)
+    time = Time.now
+    result = DatabaseConnection.query(
+      "INSERT INTO peeps (content, timestamp) VALUES($1, $2) RETURNING id, content, timestamp;",
+      [content, time]
+    )
+    Peep.new(id: result[0]['id'], content: result[0]['content'], timestamp: result[0]['timestamp'])
+  end
+
+  def self.all
+    result = DatabaseConnection.query("SELECT * FROM peeps")
```

Comment:
You record timestamp when inserting into the peeps database, you could have maybe used this to return your peeps in reverse chronological order rather than calling reverse in your controller to keep more logic in the model. I'm not sure if that is an improvement to what you have done though to be honest, this implementation would work every time too.